lib/spack/spack/schema/config.py

```diff
@@ -94,6 +94,14 @@
             "additional_external_search_paths": {"type": "array", "items": {"type": "string"}},
             "binary_index_ttl": {"type": "integer", "minimum": 0},
             "aliases": {"type": "object", "patternProperties": {r"\w[\w-]*": {"type": "string"}}},
+            "includes": {
+                "type": "array",
+                "default": [],
+                "items": {
```

Comment:
Should also allow a string to be an element of items, to match existing env include. e.g.:

```yaml
include:
- path/to/first/config
- path/to/second/config
```

Would be the same as:
```yaml
include:
- path: path/to/first/config
- path: path/to/second/config
```